### DISCUSSION TITLE
Nominate @robdimsdale to a Stacks maintainer

### DISCUSSION THREAD

**Discussion by sophiewigmore** [paketo-buildpacks/feedback#156] [Stacks Team]
@robdimsdale has contributed a significant amount of work to support the [new Jammy Stacks](https://github.com/paketo-buildpacks/jammy-tiny-stack). This work has introduced a lot of new tooling and stacks-related automation which will eventually be used in all Paketo Stack repos. He has also contributed code review, and provided opinions on a number of related issues and RFCs. His work should be recognized with a maintainership role, so that he can continue to inform the direction that our stacks go in and help Paketo to ship secure stacks.

https://github.com/orgs/paketo-buildpacks/teams/stacks-maintainers can I get a 👍 / 👎 ?

**Comment by ryanmoran** [comment]
Congrats @robdimsdale on becoming a Stacks Maintainer!! 🎉
